app/src/main/java/com/example/nigeriatelemedicineapp/api/ApiManager.kt

```diff
@@ -22,9 +23,12 @@ class ApiManager {
 
     fun createService()
     {
+        val OkHttpClient= OkHttpInterceptor().getClient("admin","Admin123")
```

Comment:
Let's move these values into static final class variables. Eventually, they should probably be configuration values, but I know we're still figuring out how we want to structure our configs.

Reply:
@isears  Solved